Assignment3/modules.rb

```diff
@@ -0,0 +1,57 @@
+#Module File for Assignment3
+require_relative 'data'
+#Input Module for Country Name
+module GetCountry
+  def self.GetCname
+    puts "Enter Country Name"
+    cname = gets.chomp.downcase
+    return  cname
+  end
+end
+#Module to find Country name in Data
+module FindCountry
+  def self.FindCname
+    cname = GetCountry.GetCname
+    count = CountryData.Country.find {|x| x[:Name] == cname}
+    if count != nil
+      return count
+    else
+      puts "Country Not Found in our Database"
+    end 
+  end
+end
+#Module to find Country is eligible for Loan or not
+module LoanEligibility
+  def self.GetLoan
+    country = FindCountry.FindCname
+    if country != nil and country[:GDP] > 4_00_00_00_000
```

Comment:
No need to check if its nil, simply you can do this:

`if country && country[:GDP] > 4_00_00_00_000`